src/webdriver_kaifuku/__init__.py

```diff
@@ -29,7 +29,7 @@ class BrowserFactory(object):
     webdriver_class = attr.ib()
     browser_kwargs = attr.ib()
 
-    def __attr_post_init__(self):
+    def __attrs_post_init__(self):
```

Comment:
@RonnyPfannschmidt funny story, these methods weren't getting executed, because of the name.

http://www.attrs.org/en/stable/examples.html
```
Sometimes, you want to have your class’s __init__ method do more than just the initialization, validation, etc. that gets done for you automatically when using @attr.s. To do this, just define a __attrs_post_init__ method in your class. It will get called at the end of the generated __init__ method.
```

Reply:
good and fun catch